spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -28,6 +28,7 @@ class { 'foreman::cli':
         include foreman::cli::tasks
         include foreman::cli::templates
         include foreman::cli::webhooks
+        include foreman::cli::rh_cloud
```

Comment:
This is only packaged for Red Hat, so I'd expect it to be similar to azure, kubevirt and openscap. However, it's even only packaged for Katello, whcih we don't set up. In that sense I suppose it's similar to `foreman::cli::katello` that we don't have acceptance tests for it. Just unit tests.